### DISCUSSION TITLE
访问google为什么有时抽风?

### DISCUSSION THREAD

**Discussion by wanghuibin0** [v2fly/v2ray-core#1704] [Q&A]
v2ray版本是4.44.0
现象大概是这样的:
访问google时好时不好,访问youtube一直都正常的.
而且很奇怪,当我访问google失败的时候,我立马去访问youtube,youtube是可以访问的.
google无法访问的时候,过一会却又好了.

我看access.log error.log,也没有发现对应的访问失败的记录.

请问小伙伴,我这情况是什么原因呢?或者说如何排查呢?

**Comment by littleghost2016** [comment]
我k2p有时候的现象和你相反，访问youtube会加载很长时间，这时候访问一下google，两边就都能加载出来了

**Reply by iscmd** [reply]
请问怎么访问google？手上没有工具

**Reply by wanghuibin0** [reply]
就用浏览器访问啊

**Comment by TibbersDriveMustang** [comment]
same here

**Comment by wanghuibin0** [comment]
有人知道吗?

**Comment by ququ0755** [comment]
可能是版本问题
